lib/yabeda/rails/railtie.rb

```diff
@@ -18,9 +18,13 @@ def puma_server?
       def unicorn_server?
         ::Rails.const_defined?("Unicorn::Launcher")
       end
+      
```

Comment:
Can you please remove this trailing whitespace?

It can be done automatically by running `bundle exec rubocop --autocorrect-all`

Reply:
Done and sorry for the late response. I got carried away by other things.